config/config.go

```diff
@@ -72,6 +72,8 @@ var (
 		HoustonDialTimeout:   newCfg("houston.dial_timeout", "10"),
 		HoustonSkipVerifyTLS: newCfg("houston.skip_verify_tls", "false"),
 		SkipParse:            newCfg("skip_parse", "false"),
+		Interactive:          newCfg("interactive", "false"),
+		PageSize:             newCfg("page_size", "100"),
```

Comment:
isn't the default page size too large for any terminal? how about 20?

Reply:
Yes, we can make that 20 default, anyhow user has the ability to set that.